### ISSUE TITLE
Add ability to handle game events with... scarpet game events.

### ISSUE THREAD

**Issue by gnembon** [gnembon/fabric-carpet#602]
one with skulk sensor

**Comment by gnembon** [comment]
Ok, will wait on that since mappings are in a terrible state around game events and the skulk sensi

**Comment by Ghoulboy78** [comment]
nice skulk sensi

**Comment by Ghoulboy78** [comment]
yo @gnembon would this mean that instead of `__on_stuff_happen(args)->function;` event, you would have `handle_event('stuff_happen','whatever_function', args);`? I personally wouldnt mind either, both seem like valid solutions to me tbh, tho ig in my reimplementation of scarpet I would like to implement the latter as opposed to the former. Just saying...

**Comment by altrisi** [comment]
No, (although I would also prefer explicit declaration), I think this means handling the events that the skulk sensor can detect with Scarpet.
